### ISSUE TITLE
stable enough to package new version to debian?

### ISSUE THREAD

**Issue by rogers0** [xtaci/kcp-go#32]
Dear Daniel,

Thanks for your work!

v2.14+ gets into Debian recently: https://tracker.debian.org/pkg/golang-github-xtaci-kcp

And I found version of the upstream is already v3.5, seems get improved a lot lately.
My question is is the latest commit stable enough to package into Debian?
or tag v3.5 is better?
What do you think?

Cheers,
Roger

**Comment by xtaci** [comment]
yes, v3.5 is stable, especially the overall memory usage is much lower compared to 2.x releases, and it can handle much more connections on the server-side.

**Comment by xtaci** [comment]
@rogers0 some weird leak just found  in v3.5 , I think we need some delay.
https://github.com/xtaci/kcptun/issues/415

**Comment by rogers0** [comment]
@xtaci thanks for getting me informed.
actually I have another issue to handle. I find latest golang-github-xtaci-kcp, which depends on golang-github-klauspost-reedsolomon-dev, uses a new function that debian package of golang-github-klauspost-reedsolomon-dev hasn't supported yet.
So I need to update golang-github-klauspost-reedsolomon-dev package first.
It may take 1-2 weeks, because I need to contact the maintainer. So no hurry for your side. Thank you!

**Comment by xtaci** [comment]
https://github.com/xtaci/kcp-go/releases/tag/v3.6 is expected to package into Debian if nothing particular happens in next 7 days.

**Comment by xtaci** [comment]
@rogers0 all the leaks has fixed,  kcp-goversion v3.7 and kcptun version v20170315 are the most stable versions ever,  I suggest to package into Debian.

**Comment by rogers0** [comment]
@xtaci thanks for your quick response!

**Comment by rogers0** [comment]
Since no other one offer the help to make new version of golang-github-klauspost-reedsolomon, I did the work myself and it's in RFS stage:
- https://bugs.debian.org/858557

After installed the updated golang-github-klauspost-reedsolomon-dev/1.3-1~exp1, now I can finally build your new kcp-go package. (I tried both v3.7 and v3.9)
So now my question is which version is better to package? v3.7, v3.9 or latest commit?
Thanks!
